pkg/resources/saml2_integration.go

```diff
@@ -180,54 +201,184 @@ func ImportSaml2Integration(ctx context.Context, d *schema.ResourceData, meta an
 
 	integration, err := client.SecurityIntegrations.ShowByID(ctx, id)
 	if err != nil {
-		if errors.Is(err, sdk.ErrObjectNotFound) {
```

Comment:
Why is it removed?

Reply:
Hmm, isn't it that if we throw the error here it won't import this object (and the setId("") is obsolete)?